tasks/config_save.rb

```diff
@@ -1,26 +1,32 @@
 #!/opt/puppetlabs/puppet/bin/ruby
-require_relative '../lib/puppet/util/task_helper'
-task = Puppet::Util::TaskHelper.new('cisco_ios')
+require 'puppet'
 
-result = {}
+require_relative "../../ruby_task_helper/files/task_helper.rb"
 
-unless Puppet.settings.global_defaults_initialized?
-  Puppet.initialize_settings
-end
+class ConfigSave < TaskHelper
+  def task(_params, remote, **kwargs)
+    unless Puppet.settings.global_defaults_initialized?
+      Puppet.initialize_settings
+    end
 
-begin
-  rtn = task.transport.running_config_save
-  result[:status]  = 'success'
-  result[:results] = "running-config saved to startup-config: #{rtn}"
-rescue StandardError => e
-  result[:_error] = {
-    msg: e.message,
-    kind: 'puppetlabs/cisco_ios',
-    details: {
-      class: e.class.to_s,
-      backtrace: e.backtrace,
-    },
-  }
+    result = {}
+    begin
+      rtn = remote.transport.running_config_save
+    rescue StandardError => e
```

Comment:
https://github.com/puppetlabs/puppetlabs-ruby_task_helper/blob/77713a72d76c2e92c6d3a3f89fbc9bb81b1c2cbc/files/task_helper.rb#L59 already handles `StandardError`, no need to repeat here. That'll save a chunk of code here.